packages/millicast-sdk/src/View.js

```diff
@@ -38,11 +38,10 @@ const connectOptions = {
  * @constructor
  * @param {String} streamName - Deprecated: Millicast existing stream name.
  * @param {tokenGeneratorCallback} tokenGenerator - Callback function executed when a new token is needed.
- * @param {HTMLMediaElement} [mediaElement=null] - Target HTML media element to mount stream.
  * @param {Boolean} [autoReconnect=true] - Enable auto reconnect to stream.
  */
 export default class View extends BaseWebRTC {
-  constructor (streamName, tokenGenerator, mediaElement = null, autoReconnect = true) {
+  constructor (streamName, tokenGenerator, autoReconnect = true) {
```

Comment:
Like `streamName`, can we mark `mediaElement` as deprecated and handle the same way? Otherwise its a breaking change for players if the constructor of the view class all of a sudden doesn't receive one property.

And handle it the same way as we handle streamName

Reply:
done